mergify_engine/constants.py

```diff
@@ -16,3 +16,55 @@
 
 MERGE_QUEUE_BRANCH_PREFIX = "mergify/merge-queue"
 MERGE_QUEUE_SUMMARY_NAME = "Queue: Embarked in merge train"
+
+MERGIFY_OPENSOURCE_SPONSOR_DOC = (
+    "<hr />\n"
+    ":sparkling_heart:&nbsp;&nbsp;Mergify is proud to provide this service "
+    "for free to open source projects.\n\n"
+    ":rocket:&nbsp;&nbsp;You can help us by [becoming a sponsor](/sponsors/Mergifyio)!\n"
+)
+MERGIFY_MERGE_QUEUE_PULL_REQUEST_DOC = """
+
+More informations about Mergify merge queue can be found in the [documentation](https://docs.mergify.io/actions/queue.html).
+
+<details>
+<summary>Mergify commands</summary>
+
+<br />
+
+You can also trigger Mergify actions by commenting on this pull request:
+
+- `@Mergifyio refresh` will re-evaluate the queue rules
+
+Additionally, on Mergify [dashboard](https://dashboard.mergify.io/) you can:
+
+- look at your merge queues
```

Comment:
not sure it's a good thing since this is only for strict mode :(

Reply:
We need to do the same for queue action no?

Reply:
Sure! My point is that this is not available (yet), not sure about saying this here and now.